spec/javascript/components/molecules/StatusItem.test.js

```diff
@@ -50,4 +50,10 @@ describe('StatusItem', () => {
     const wrapper = getWrapper(mockStatusItem({ status: 'Lease Signed', substatus: null }))
     expect(wrapper).toMatchSnapshot()
   })
+
+  test('should format timestamps as expected', () => {
+    const wrapper = mount(<StatusItem statusItem={mockStatusItem({ status: 'Lease Signed', substatus: null })} />)
+    const timestamp = wrapper.find('StatusDate').text()
```

Comment:
I'm surprised that you can search for a component name using mount! I guess I thought it would all be div and class selectors